### DISCUSSION TITLE
[FIXED] Problem allready fixed

### DISCUSSION THREAD

**Discussion by ghost** [vilhelmprytz/pterodactyl-installer#245] [Q&A]
Hey, 
I have not much experience with linux, so maybe you can help me.
**_I wanted to install it and at the end this come_**:

Created Rust

Seeded:  Database\Seeders\EggSeeder (190.38ms)
Database seeding completed successfully.

In Model.php line 63:

  Data integrity exception encountered while performing database write operation. {"username":["T
  he username must start and end with alpha-numeric characters and\n                contain only
  letters, numbers, dashes, underscores, and periods."]}

**_On the Website:_**

Welcome to nginx!
If you see this page, the nginx web server is successfully installed and working. Further configuration is required.

For online documentation and support please refer to nginx.org.
Commercial support is available at nginx.com.

Thank you for using nginx.

**Hope you can help me, thanks!
Best wishes noqcz**

**Comment by Linux123123** [comment]
The username you chose can't be used because
```The username must start and end with alpha-numeric characters and contain only letters, numbers, dashes, underscores, and periods.```

**Comment by ghost** [comment]
Thanks for the answer, I fixed it.
